py/jpy/src/main/c/jpy_jtype.c

```diff
@@ -455,7 +455,7 @@ int JType_CreateJavaNumberFromPythonInt(JNIEnv* jenv, JPy_JType* type, PyObject*
     j = JPy_AS_JLONG(pyArg);
     i = (int) j;
     s = (short) j;
-    b = (char) j;
+    b = (signed char) j;
```

Comment:
Shouldn't we declare `b` with the same type? Do we need a similar fix in anything else?

Reply:
don't think that's needed, but it is good practice anyways. btw, char is the only integer type in C that can be either signed or unsigned by default, all the other types: short, int, long long are signed by default.

Reply:
I have already done a quick scan/audit of the code base, this appears the only place where we use C char type to convert to Java byte.